packages/theme/src/types.ts

```diff
@@ -72,18 +72,33 @@ type PaletteAdditions = {
 export type BackstagePalette = Palette & PaletteAdditions;
 export type BackstagePaletteOptions = PaletteOptions & PaletteAdditions;
 
+export type pageThemeOptions = {
```

Comment:
Should be PascalCase. I'm thinking options isn't quite what this is too, and might conflict with functions for creating page themes. Perhaps `PageThemeSelector` instead?